presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,50 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result of ``last_day_of_month`` function for first day of month. (:issue:`2452`)
+* Handle common disk failures during the spill. When on of disks got broken, it Presto will use another
+  one without failing the query. (:issue:`2444`)
+* Reduce number of metadata queries when querying ``information_schema``. (:issue:`2488`)
```

Comment:
end-user visible impact is "faster" or "improve performance"; maybe also "reduce metastore load":
```
* Improve performance and reduce load on external systems when querying ``information_schema``. (:issue:`2488`)
```